docs/guide/connected-apps.md

```diff
@@ -0,0 +1,31 @@
+title: Connected Apps Guide
+description: Connected Apps  is a data pivoting feature and available in all solutions. You can connect any two Apps you have access to, regardless of their type.
+
+Connected Apps  is a data pivoting feature and available in all solutions. You can connect any two Apps you have access to, regardless of their type.
```

Comment:
Space
Also, what is "data pivoting feature"?  Do we have other examples of that?  If so, should we be thinking about User Guide > Data Pivoting page?  Does hyperlinking in Events we are doing qualify as a data pivoting feature?

Reply:
I mean hyperlinking could be also data pivoting feature yes. I described this one as data pivoting feature because it is what we say for Split Screen here https://sematext.com/docs/guide/split-screen/ and Connected Apps and Split Screen features are highly related with each other. And these 2 are listed under User Guide